policy/modules/contrib/nbdkit.fc

```diff
@@ -0,0 +1,5 @@
+HOME_DIR/tmp(/.*)?        gen_context(system_u:object_r:nbdkit_home_t,s0)
+
+/usr/sbin/nbdkit	 --    gen_context(system_u:object_r:nbdkit_exec_t,s0)
+
+/usr/lib/systemd/system/nbdkit.*	gen_context(system_u:object_r:nbdkit_unit_file_t,s0)
```

Comment:
I cannot find any services like this in Fedora.

Reply:
It is based on the comment [20](https://bugzilla.redhat.com/show_bug.cgi?id=2182505#c20)